src/containers/App.tsx

```diff
@@ -187,13 +195,27 @@ const Content = (): JSX.Element => {
 
     const isOnTavle = !['/privacy', '/tavler'].includes(location.pathname)
 
+    const boardId = useRouteMatch<{ documentId: string }>('/t/:documentId')
+        ?.params?.documentId
+
     const Dashboard = settings[0]
         ? getDashboardComponent(settings[0].dashboard)
         : (): null => null
 
+    const [isRotated, setIsRotated] = useState(false)
+
     useEffect(() => {
         updateManifest(window.location.href, window.location.origin)
-    }, [location.pathname])
+        if (window.location.href.includes('/t/')) {
```

Comment:
Skulle vi kanskje laget en hjelpemetode her for å avgjøre om det er en tavla man er inne på eller ikke?

Reply:
Enig at vi burde ha større klarhet i hva den gjør men usikker på om vi trenger å lage en hjelpemetode. Forandret til å bruke en react-router hook og ga den et mer beskrivende navn, hva tenker du om det?